build_docs.py

```diff
@@ -98,8 +159,9 @@ def build_one(version, isdev, quick, sphinxbuild, build_root, www_root,
     maketarget = "autobuild-" + ("dev" if isdev else "stable") + ("-html" if quick else "")
     logging.info("Running make %s", maketarget)
     logname = os.path.basename(checkout) + ".log"
-    shell_out("cd Doc; make SPHINXBUILD=%s %s >> %s 2>&1" %
-              (sphinxbuild, maketarget, os.path.join(log_directory, logname)))
+    shell_out("cd Doc; make SPHINXBUILD=%s SPHINXOPTS='%s' %s >> %s 2>&1" %
+              (sphinxbuild, sphinxopts, maketarget,
+               os.path.join(log_directory, logname)))
```

Comment:
Ok, it perfectly makes sense. But it would help if someone could give you access to the logs since you seem to be one of the people working on this project :-)

@ewdurbin: Can you please give access to Julien to the logs? Or maybe send him logs on demand? Thanks.

FYI I proposed to give the commit bit to @JulienPalard on this project :-)
https://mail.python.org/pipermail/python-committers/2017-June/004583.html